### ISSUE TITLE
Add an explicit Jev fast path for bounded Gemma cognition

### ISSUE THREAD

**Issue by rinsakamo** [rinsakamo/relay-self#255]
## Purpose

Add the smallest explicit Jev/System One execution path for existing `BOUNDED` RelayEngine cognition so current Gemma 4 12B experiments can avoid text/JSON generation when a finite-choice readout is sufficient.

This is performance work intended to preserve the existing RelayEngine semantic boundary. It does not add a new cognition owner, Skill, model, substrate selector, or hidden fallback.

Refs #101.
Refs #88.

## Current baseline

Current `LlamaCppRelayProvider` realizes `BOUNDED` through `/v1/chat/completions` with strict JSON generation:

```text
BoundedChoiceRequest
  -> prompt evaluation
  -> generated JSON
  -> parse
  -> ProviderDecision
```

Recorded live Gemma 4 12B Q4_K_M examples are roughly one second for ~440 prompt tokens plus 12-18 generated tokens (#140 / #190).

The external Jev llama.cpp branch exposes `/v1/systemone`, which performs one prompt evaluation, reads finite-choice next-token logits, and emits zero model output tokens.

## Minimum implementation

Keep one `LlamaCppRelayProvider`.

- existing chat-completions endpoint remains responsible for THINK / OPEN;
- add an explicit optional System One endpoint for BOUNDED only;
- when configured, BOUNDED sends one finite-choice Jev request and does not call chat completions;
- no automatic fallback from Jev to generated bounded cognition;
- preserve `ProviderDecision`, `ProviderCallFacts`, admissible choice validation, and RelayEngine THINK escalation;
- represent cognitive insufficiency as an explicit internal finite choice rather than inventing a new confidence threshold in this transaction;
- keep provenance-bearing state in the request;
- record Jev input/output token facts when supplied.

## Non-goals

Do not:
- make Jev a semantic owner;
- add a second model;
- add a provider registry/router;
- use Jev for THINK or OPEN;
- claim calibration for the current Q4_K_M artifact;
- remove the generated BOUNDED baseline before matched measurement;
- silently fall back when the Jev endpoint is absent or malformed.

## Verification

Deterministic tests should prove:
- exact Jev request shape;
- resolved finite choice;
- explicit unresolved choice;
- malformed/inadmissible answer fails as provider protocol error;
- one Jev BOUNDED call can still escalate to one chat THINK call when unresolved;
- existing chat BOUNDED / THINK / OPEN tests remain green.

Physical qualification and canonicalization are separate follow-up evidence.

**Comment by rinsakamo** [comment]
## Implementation complete

PR #256 was squash-merged on current main.

Merged authority:

```text
main = 264c76b35820c0b399e3b082519cc4ca4cfbf1e4
tree = 92a559defd06808cdbe303630e2c4b3d5aa6d8d9
open PRs targeting main = 0
```

Exact-head CI on PR #256 passed:

- repository-contracts
- pytest
- lint
- mineflayer-adapter

Implemented boundary:

```text
one LlamaCppRelayProvider
  BOUNDED + explicit systemone_endpoint
    -> /v1/systemone
    -> finite-choice logits readout
    -> RESOLVED | explicit UNRESOLVED
  UNRESOLVED
    -> existing explicit THINK
    -> /v1/chat/completions
  OPEN
    -> /v1/chat/completions
```

No confidence threshold, hidden fallback, second model, substrate router, or new semantic owner was added.

The existing generated BOUNDED path remains available as an explicit measurement baseline. This closes the implementation transaction only; no physical Jev latency/quality claim is made here.
